recipes/bgen-reader/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "bgen-reader" %}
+{% set version = "1.1.2" %}
+{% set sha256 = "ba1d845cac4c14930a1f1270829b2fc3cbf259f383c8067e8e83b633a8c83afc" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win and py27]
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - bgen >=1.1.4
+    - cffi >=1.11.4
+    - pip
+    - pytest-runner >=3.0
+    - python
+    - setuptools >=38.4.0
+    - toolchain
+  run:
+    - bgen >=1.1.4
+    - cffi >=1.11.4
+    - dask >=0.16.1
+    - pandas >=0.22.0
+    - pytest >=3.3.2
+    - pytest-pep8 >=1.0.6
+    - python
+    - scipy >=1.0.1
+    - toolz >=0.9.0
+    - tqdm >=4.19.5
+
+test:
+  requires:
+    - pytest
+    - pytest-pep8
+  imports:
+    - pandas
```

Comment:
I don't think you need to import `pandas` here, but since this is my only issue, just consider fixing it in the feedstock.